src/Header/HeaderWrap.php

```diff
@@ -90,7 +90,7 @@ protected static function wrapStructuredHeader($value, StructuredInterface $head
      */
     public static function mimeEncodeValue($value, $encoding, $lineLength = 998)
     {
-        return Mime::encodeQuotedPrintableHeader($value, $encoding, $lineLength, Headers::EOL);
+        return Mime::encodeBase64Header($value, $encoding, $lineLength, Headers::EOL);
```

Comment:
i agree this is big, but quoted printable encoding has coner case that i don't know how to handle (comma ...)
if it's too big i can put it in another PR